### PR TITLE
Test/streetcode Fact get by Id

### PR DESCRIPTION
**Pull request by yaninavoda**
## Summary of issue

Tests for Streetcode/Fact GetFactByIdHandler

## Summary of change

Added 7 tests for positive and negative scenarios

### CODE REVIEW

**Review by HlibBondarev — APPROVED** [review]
Ok

File: Streetcode/Streetcode.WebApi/Program.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 await app.ApplyMigrations();
 
-// await app.SeedDataAsync(); // uncomment for seeding data in local
+await app.SeedDataAsync(); // uncomment for seeding data in local
```
Comment (vladtymo) [inline Streetcode/Streetcode.WebApi/Program.cs:30]: Revert this changes

Comment (yaninavoda) [reply Streetcode/Streetcode.WebApi/Program.cs]: Done. Commented out the line with SeedingData

**Review by OlhaPivniuk — APPROVED** [review]
